src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets

```diff
@@ -123,6 +123,13 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!-- Determine PlatformTarget (if not already set) from runtime identifier. -->
   <Choose>
+    <!-- For all targets other than .NETFramework, set $(PlatformTarget) to AnyCPU. -->
+    <When Condition="'$(PlatformTarget)' == '' and '$(TargetFrameworkIdentifier)' != '.NETFramework' and '$(UseNativeCode)' != 'true'">
```

Comment:
Is `UseNativeCode` a pre-existing property or is this a new property introduced by this PR?

If it is a new property introduced to disable the new behavior, it should have a specific name, something like `UseSpecificPlatformTargetIfPossible`.